Hyperbolic_triangulation_2/include/CGAL/Hyperbolic_Delaunay_triangulation_traits_2.h

```diff
@@ -376,8 +376,8 @@ class Construct_hyperbolic_bisector_2
   {
     CGAL_triangulation_precondition((_gt.orientation_2_object()(p, q, r) == ON_POSITIVE_SIDE) &&
                                     (_gt.orientation_2_object()(p, s, q) == ON_POSITIVE_SIDE));
-    CGAL_triangulation_precondition((_gt.side_of_oriented_circle_2_object()(p, q, r,s) == ON_NEGATIVE_SIDE) &&
-                                    (_gt.side_of_oriented_circle_2_object()(p, s, q, r) == ON_NEGATIVE_SIDE));
```

Comment:
This precondition asserts that the faces `(p,q,r)` does not contain `s`, and the face `(p,s,q)` does not contain `r`, which is satisfied if the two faces `(p,q,r)` and `(p,s,q)` are Delaunay. If the assertion is failing, I feel like more investigation is needed, instead of just commenting out the check altogether.

Reply:
Indeed, just commenting out is dangerous. 
I am a bit surprised that, if the assertion is problematic, this has not been detected by the test-suite...

Reply:
@switzel Can you comment on why you removed that precondition?

Reply:
Ping @switzel!